src/MeshKernelNET/Native/MeshKernelDll.cs

```diff
@@ -1742,6 +1742,13 @@ internal static extern int Mesh2dTranslate([In] int meshKernelId,
         [DllImport(MeshKernelDllName, EntryPoint = "mkernel_get_inner_outer_separator", CallingConvention = CallingConvention.Cdecl)]
         internal static extern double GetInnerOuterSeparator();
 
+        /// <summary>
+        /// Gets the integer (int) value used in the back-end library missing value and null identifier
+        /// </summary>
+        /// <returns>The int missing value used in mesh kernel</returns>
+        [DllImport(MeshKernelDllName, EntryPoint = "mkernel_get_null_identifier", CallingConvention = CallingConvention.Cdecl)]
+        internal static extern double GetNullIdentifier();

```

Comment:
The documentation says it's an int value, but the function returns a double. This is confusing. Either adjust the summary, or the function's return type. It would be helpful to explain in some more detail what these null identifiers and missing value tokens are used for. So far I only knew the geometryseparator and inner-outer separator, and the missing-value token-value used for e.g. orthogonality and smoothness.

Reply:
this function is now removed, not needed for the first iteration of undo/redo